help/en/releasenotes/current-draft-note.shtml

```diff
@@ -404,6 +404,19 @@
     <h3>LogixNG</h3>
         <a id="LogixNG" name="LogixNG"></a>
         <ul>
+          <li>The ConditionalNG editor now have better support for the keyboard.
+          Enable the <i>Preferences -> LogixNG</i> option <i>Highlight row in
+          ConditionalNG editor</i>.
+          You can then use the arrow keys.
```

Comment:
There needs to be a phrase such as "For macOS, use Cmd instead of Ctrl."

Reply:
Fixed.

Would it be better to have this line:
`The ConditionalNG editor now have better support for the keyboard.`
as:
`The ConditionalNG editor has now better support for the keyboard.`
?

Reply:
Here is my recommendation for the release note.
```
            <ul>
              <li>Keyboard support has been added to the ConditionalNG editor.
                <ul>
                  <li>
                    When <i>Preferences -> LogixNG</i> option <i>Highlight row in
                    ConditionalNG editor</i> is enabled, use the arrow keys to move to the
                    desired row and then use one of the shortcut keys.
                  </li>
                  <li>
                    Except for the N shortcut key, the other shortcut keys are available in
                    the right click popup menu.
                  </li>
                  <li>Shortcut keys:
                    <ul>
                      <li>Ctrl + N: Create new</li>
                      <li>Ctrl + E: Edit</li>
                      <li>Ctrl + R: Remove</li>
                      <li>Ctrl + X: Cut</li>
                      <li>Ctrl + C: Copy</li>
                      <li>Ctrl + V: Paste</li>
                    </ul>
                  </li>
                  <li>
                    For macOS, use Cmd instead of Ctrl.
                  </li>
                </ul>
              </li>
            </ul>
```

Which looks like:

![Screen Shot 2022-09-18 at 9 40 22 AM](https://user-images.githubusercontent.com/17280992/190912732-d625b6b2-e619-4d85-ad08-1d676ad62cc5.png)